### ISSUE TITLE
[verify] *strategy-b*: tx on multiple, rx on single ch

### ISSUE THREAD

**Issue by nilay994** [nilay994/PercEvite#1]
https://github.com/nilay994/PercEvite/blob/e085100b1a52515e8b24453bd13e9e7b3a8c1894/ESP32_firmware/Only_BS/Only_BS.ino#L60-L67

@ingframin, the best throughput (inter-drone data exchange) can be acheived by strategyB right? Do you think I have done it correctly? 

I am currently missing packets between drones. Am I scanning too slow? Am I changing SSID too slow? Not sure. Currently monitoring via Wireshark, but taking longer - since my packets are encoded in hex :p

![image](https://user-images.githubusercontent.com/22093651/79635421-18779880-8171-11ea-8d64-7319d06540a4.png)

**Comment by ingframin** [comment]
If you use the code "as is" you should be able to receive 50% of the sent packets which corresponds to an average of ~8 packets/s.
This only works scanning a single channel. Wireshark probably does a full scan which takes a couple of seconds.
Which kind of throughput are you expecting exactly?
Did you check if you get any memory error?

**Comment by nilay994** [comment]
Yes! I was averaging 50% of the sent packets. I've been using the same `OnlyBS` firmware on both ESP32's. 

1. Would capturing via Wireshark give me more packets than capturing via another ESP32 that is running OnlyBS?

I was testing the drones in a small room, the time of possible collision from t0 = 5s. I want the reporting of pos and vel of the robot to be atleast 15 times in 6s, for it to be reliably tested in a small room. I think if data is exchanged bi-directionally at 15/5 = 3 Hz.. then I would be fine. 

2. According to you, `OnlyBS` promises bi-directional ssid exchange of 10 Hz right?

**Comment by ingframin** [comment]
1. It depends on your wireless card and how long it takes to scan.
The problem with a normal wi-fi is latency. You get more packets because you scan for longer but you can only get the results at the end of the scan which may last for few seconds.

2. OnlyBS gives you on average 8.33Hz. You can try to tweak it to go a bit faster but I never managed to go very high without memory errors.
